package.json

```diff
@@ -28,7 +28,7 @@
     "@storybook/react": "^8.2.5",
     "@storybook/react-vite": "^8.2.5",
     "@storybook/test": "^8.2.5",
-    "@tscircuit/core": "^0.0.390",
+    "@tscircuit/core": "https://pkg.pr.new/tscircuit/core/@tscircuit/core@a7878e1",
```

Comment:
this is correct, after you merge to core as a courtesy you should come back and update this (shouldn't do anything bad)